### DISCUSSION TITLE
Last command and WezTerm version copied to line when opening tmux

### DISCUSSION THREAD

**Discussion by TannerGabriel** [wezterm/wezterm#4139] [Q&A]
Hi, when I try to use tmux inside of wezterm (`tmux attach` or `tmux new`) the last command and the term `>|WezTerm 20230712-072601-f4abf8fd` is copied to the current line, which I have to delete before entering any other commands. I tried the same in the default terminal and it worked fine. Are there any configurations I need to make to get this working?

## Example

When entering `tmux new` the current line contains the following:

```
tmux new>|WezTerm 20230712-072601-f4abf8fd
```

Any help is highly appreciated—big thanks for the fantastic terminal.

**Comment by wez** [comment]
that looks like a response to a terminal version query. Is something in your shell or tmux startup trying to probe the terminal and not waiting long enough for the response?

**Reply by wez** [reply]
Possibly relevant:
* https://github.com/wez/wezterm/issues/3357

**Reply by TannerGabriel** [reply]
I thought it was related to the [following issue](https://github.com/microsoft/WSL/issues/5931) and added the suggested lines, which didn't fix the problem immediately. After reinstalling tmux and all my installed plugins the problem seems to be solved (I cannot reproduce it currently). 

```
set -sg escape-time 10
```
